build.gradle

```diff
@@ -107,41 +119,55 @@ updatedBallerinaPackages = Utils.findUpdatedBallerinaPackages(openApiPackageDirP
 
 // Temporarily added.
 task codeBuild {
-    // Pulling slalpha5 distribution if not available
-    if (project.hasProperty("remote") ) {
-        if (remote) {
-            exec {
-                commandLine 'sh', '-c', "${ballerinaDistributionPath}/bin/bal dist pull slalpha5"
-            }
+    // if (project.hasProperty("remote") ) {
+        // Pulling slalpha5 distribution if not available
+        // if (remote) {
+        //     exec {
+        //         commandLine 'sh', '-c', "${ballerinaDistributionPath}/bin/bal dist pull slalpha5"
+        //     }
+
+        //     exec {
+        //         commandLine 'sh', '-c', "rm -rf ${ballerinaDistributionPath}/bin/../dependencies/jdk-11.0.8+10-jre"
+        //     }
+
+        //     // Changing the build distribution to SL Alpha5
+        //     exec {
+        //         commandLine 'sh', '-c', "${ballerinaDistributionPath}/bin/bal dist use slalpha5"
+        //     }
+        // }
+    // }
```

Comment:
Any reason for having commented code?

Reply:
Yes, these commented code is related to code generation, and commented out at the moment as a temporary fix untill the issues in the OpenAPI build tool get fixes.